aggregator/src/config.rs

```diff
@@ -224,8 +224,8 @@ pub struct JobDriverConfig {
         alias = "retry_max_interval_millis"
     )]
     pub retry_max_interval_ms: u64,
-    /// The maximum elapsed time, in milliseconds, to wait before giving up on retrying a retryable
-    /// HTTP request.
+    /// The maximum retry time, in milliseconds, to wait before giving up on retrying a retryable
+    /// HTTP request. Note this is the sum of the retry intervals, not wall-clock time.
     #[serde(
         default = "JobDriverConfig::default_retry_max_elapsed_time_ms",
         alias = "retry_max_elapsed_time_millis"
```

Comment:
As noted in the top-comment, I am of two minds whether we should rename this. 
```suggestion
        default = "JobDriverConfig::default_retry_max_delay_ms",
        alias = "retry_max_delay_millis"
```

Reply:
Renaming config options is kind of a chore, and I think the actual difference between the sum of retry intervals and the wall time wouldn't be a big deal anyway. I don't think we ever made particularly strong promises about the accuracy of the max elapsed time.